src/js/components/ServiceSearchFilter.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FilterInputText from './FilterInputText';
+import QueryParamsMixin from '../mixins/QueryParamsMixin';
+import ServiceFilterTypes from '../constants/ServiceFilterTypes';
+
+const METHODS_TO_BIND = ['setSearchString'];
+
+class ServiceSearchFilter extends mixin(QueryParamsMixin) {
+  static get defaultProps() {
+    return {
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.state = {
+      searchString: ''
```

Comment:
Not with the new Class syntax. You set the initial state object (which is bound to the class) in the constructor.